artemis/modules/classifier.py

```diff
@@ -56,6 +66,10 @@ def _classify(data: str) -> TaskType:
         """
         :raises: ValueError if failed to find domain/IP
         """
+
+        if ":" in data:
```

Comment:
as mentioned below, netloc might have auth in URI, so this check is not foolproof. This would also break on ipv6.  
Either use some helper function for retrieving port or use regex for retrieving port